### PR TITLE
Work on style class

### PR DESCRIPTION
**Pull request by fcooper8472**

### CODE REVIEW

**Comment by fcooper8472** [comment]
@martinjrobins here's my initial work on the style class.  As you can see, got a little sidetracked on colours.  Not necessarily ready to merge yet: looks like some appveyor problems again, and the style class needs expanding and adding in to drawable.

**Comment by fcooper8472** [comment]
Thinking about it, `Style` (#41) might fit better in `util` than `frontend`: what do you think @martinjrobins

**Comment by martinjrobins** [comment]
yea, think `Style` is a better fit in `util`

File: src/backend/Backend.hpp [hunk 1]
```diff
@@ -122,7 +122,7 @@ struct TransformMatrix {
   }
 };
 
-enum Align {
+enum Align : unsigned int {
```
Comment (martinjrobins) [inline src/backend/Backend.hpp]: nice, didn't know you could do that with an enum

**Pull request merged**